### ISSUE TITLE
Question on hyperparameter differences (paper Table 9 vs. repo configs) for nuPlan main experiment

### ISSUE THREAD

**Issue by Mossforest** [autonomousvision/CaRL#24]
Thanks for this great work and for open-sourcing it! It's a valuable contribution to the community and has been very helpful for my own research.

I'm currently trying to reproduce the main experiment conducted by CaRL on nuPlan. I noticed that the experimental hyperparameters in `nuPlan/scripts/gym/train_multi_gpu.sh` and `nuPlan/carl_nuplan/planning/script/config/gym/default_gym.yaml` on GitHub are not entirely consistent with those provided in Section B.2, Table 9 of the paper.

I would like to ask if there are any other details on GitHub that differ from the paper and could affect the reproduction of the main nuPlan experiment, and if so, what changes would be necessary?

**Comment by DanielDauner** [comment]
Hey @Mossforest, thanks for reaching out. 

It's not ideal that the bash script, hydra config, and paper arguments aren't consistent. If I remember correctly, the `train_multi_gpu.sh` is rather a demo or single-gpu test script rather than the script to reproduce the entire training runs. We also regularly overwrote the input arguments of default config file. If in doubt, the paper parameters should be accurate.

If there are specific arguments that are missing, or there is confusion, let us know. We can also add some bash script and configs that reflect the training runs in the paper. 

Best,
Daniel

**Comment by Mossforest** [comment]
@DanielDauner Thank you very much for your quick reply! This helps m understand the current situation. Would it be possible for you to provide the config file or bash for the main nuPlan experiment as described in the paper? I would like to make sure if there's any detail settings omitted in the paper. Thanks a lot!

**Comment by DanielDauner** [comment]
I made the repository public I use for development. You can find the training config runs of the CoRL submission [here](https://github.com/DanielDauner/carl_nuplan/tree/corl_submission). Note that the `corl_submission` branch should have what you are searching for. The scripts for the long 1B training runs (with the regular and nuplan reward) are [here](https://github.com/DanielDauner/carl_nuplan/blob/corl_submission/scripts/ferranti/gym/).

I don't think that refactoring changed a lot of things. Anyway, I hope this is useful for the parameters.

**Comment by Mossforest** [comment]
Thank you once again for your assistance! I truly appreciate your help. Wishing you a wonderful day!

**Comment by DanielDauner** [comment]
You too! I will close the issue for now. 😃
